src/components/backend-ai-webui.ts

```diff
@@ -1895,7 +1898,7 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
             </div>
             <address class="full-menu">
               <small class="sidebar-footer">Lablup Inc.</small>
-              <small class="sidebar-footer" style="font-size:9px;">24.03.0-alpha.4.5862</small>
+              <small class="sidebar-footer" style="font-size:9px;">24.03.0-alpha.4.5927</small>
```

Comment:
Please revert this line.
```suggestion
              <small class="sidebar-footer" style="font-size:9px;">24.03.0-alpha.4.5862</small>
```